docs/get-started-cloud.md

```diff
@@ -9,176 +9,252 @@ weight=2
 
-## Using Docker Machine with Docker Swarm
+2. Go to the Digital Ocean console to view the new Droplet.
 
-Docker Machine can also provision [Swarm](https://github.com/docker/swarm)
-clusters. This can be used with any driver and will be secured with TLS.
+    ![Droplet in Digital Ocean created with Machine](img/ocean_droplet.png)
 
-First, create a Swarm token. Optionally, you can use another discovery service.
-See the Swarm docs for details.
+3. At the command terminal, run `docker-machine ls`.
 
-To create the token, first create a Machine. This example will use VirtualBox.
+        $ docker-machine ls
+        NAME             ACTIVE   DRIVER         STATE     URL                         SWARM
+        default          *        virtualbox     Running   tcp://192.168.99.100:2376   
+        docker-sandbox   -        digitalocean   Running   tcp://45.55.139.48:2376     
 
-    $ docker-machine create -d virtualbox local
+    Notice that the new cloud server is running but is not the active host. Our command shell is still connected to the default machine, which is currently the active host as indicated by the asterisk (*).
 
-Load the Machine configuration into your shell:
+4. Run `docker-machine env docker-sandbox` to get the environment commands for the new remote host, then run `eval` as directed to re-configure the shell to connect to `docker-sandbox`.
 
-    $ eval "$(docker-machine env local)"
+        $ docker-machine env docker-sandbox
+        export DOCKER_TLS_VERIFY="1"
+        export DOCKER_HOST="tcp://45.55.222.72:2376"
+        export DOCKER_CERT_PATH="/Users/victoriabialas/.docker/machine/machines/docker-sandbox"
+        export DOCKER_MACHINE_NAME="docker-sandbox"
+        # Run this command to configure your shell:
+        # eval "$(docker-machine env docker-sandbox)"
 
-Then run generate the token using the Swarm Docker image:
+        $ eval "$(docker-machine env docker-sandbox)"
 
-    $ docker run swarm create
-    1257e0f0bbb499b5cd04b4c9bdb2dab3
+5. Re-run `docker-machine ls` to verify that our new server is the active machine, as indicated by the asterisk (*) in the ACTIVE column.
 
-Once you have the token, you can create the cluster.
+        $ docker-machine ls
+        NAME             ACTIVE   DRIVER         STATE     URL                         SWARM
+        default          -        virtualbox     Running   tcp://192.168.99.100:2376   
+        docker-sandbox   *        digitalocean   Running   tcp://45.55.222.72:2376     
 
-### Swarm master
+6. Log in to the Droplet with the `docker-machine ssh` command.
```

Comment:
You don't need to do that, and this should be discouraged, since the purpose of machine is to (some extent) remove the need from ssh-ing into remote machines.

To check that the docker engine is working, just stay where you are (your local environment) and use docker itself.